package.json

```diff
@@ -147,5 +148,8 @@
   },
   "resolutions": {
     "cssom": "0.3.4"
+  },
+  "dependencies": {
+    "lodash": "^4.17.11"
```

Comment:
Lodash really shouldn't be needed for anything anymore. I am super opinionated about lodash and how it is overkill for most things though so this is up to others. I expressed my concerns about the deep isEqual checks in other comments